mx.sulong/mx_sulong.py

```diff
@@ -1141,6 +1148,18 @@ def checkNoHttp(args=None):
     'main-arg' : runMainArgTestCases,
 }
 
+checkCases = {
+    'gitlog' : logCheck,
```

Comment:
Can you verify that each of the code checks actually fails when it sees code violations? Do `checkoverlap` and `checkstyle` actually fail with a Python exit or just return a non-zero value? In latter case, the code check would not abort on the gate.